aliasing/api/combat.py

```diff
@@ -100,6 +100,16 @@ def __init__(self, combatant: Combatant, hidestats=True):
         self.initmod = int(self._combatant.init_skill)
         self.init = self._combatant.init
         self._update_effects()
+        # Type-specific Properties
+        self._race = None
+        self._creature_type = None
+        self._monster_name = None
+
+        if combatant.type == CombatantType.MONSTER:
+            self._creature_type = combatant._creature_type
+            self._monster_name = combatant.monster_name
+        elif combatant.type == CombatantType.PLAYER:
+            self._race = combatant.character.race
```

Comment:
thoughts on setting creature type to humanoid?

Reply:
Conflicts with a few existing races like Centaur and Satyr (fey)